src/network/networkd-dhcp-prefix-delegation.c

```diff
@@ -570,6 +572,41 @@ static int dhcp6_pd_assign_prefix(
         return 1;
 }
 
+static int dhcp6_pd_assign_prefix_on_uplink(
+                Link *link,
+                const struct in6_addr *pd_prefix,
+                uint8_t pd_prefix_len,
+                usec_t lifetime_preferred_usec,
+                usec_t lifetime_valid_usec) {
+
+        _cleanup_free_ char *buf = NULL;
+        int r;
+
+        assert(link);
+        assert(link->network);
+        assert(pd_prefix);
+
+        (void) in6_addr_prefix_to_string(pd_prefix, pd_prefix_len, &buf);
+
+        if (link->network->dhcp6_pd_announce)
+                log_link_debug(link, "Ignoring Announce= setting on upstream interface.");
```

Comment:
"upstream" → "uplink" ?

Reply:
Well,,, I use `uplink` in the variable name and comments, `"upstream interface"` in the log message or man page.
If you do not like it. I will unify that in another later commit, but I do not think it is necessary though that's not strong opinion.